components/trello/cards-due.js

```diff
@@ -0,0 +1,67 @@
+const trello = require("https://github.com/PipedreamHQ/pipedream/components/trello/trello.app.js");
+
+module.exports = {
+  name: "Cards Due",
+  description:
+    "Emits an event at a specified time before a card is due.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    trello,
+    boardId: { propDefinition: [trello, "boardId"] },
+    timeBefore: {
+      type: "integer",
+      label: "Time Before",
+      description: "How far before the due time the event should trigger.",
+      default: 5,
+    },
+    timeBeforeUnit: {
+      type: "string",
+      label: "Time Before (Unit)",
+      description: "Unit of time for Time Before.",
```

Comment:
Since you set a default above does it make sense to have a default here, as well?